app/controllers/partners/promo_campaigns_controller.rb

```diff
@@ -0,0 +1,27 @@
+include ActionView::Helpers::SanitizeHelper
+module Partners
+    class PromoCampaignsController < ApplicationController
+    def create
+        promo_campaign = PromoCampaign.create(
+            name: sanitize(params[:name]),
```

Comment:
You don't have to sanitize the input here because rails should do it by default. If you are assigning the entire params then it might make more sense to use [strong parameters](https://edgeguides.rubyonrails.org/action_controller_overview.html#strong-parameters).